### PR TITLE
Clean directory name for dataset

### PR DESCRIPTION
**Pull request by idonov8**
I tested names for paths on ` ds.add(file=f, path="./weird_name.exe/")` and it works fine.

### CODE REVIEW

File: dagshub/upload/wrapper.py [hunk 1]
```diff
@@ -153,7 +155,7 @@ def _log_upload_details(self, data, res, files):
             logger.debug(f"Response ({res.status_code})\n")
 
         if res.status_code == 200:
-            logger.info("Upload finished successfully!")
+            print("Upload finished successfully!")
```
Comment (guysmoilov) [inline dagshub/upload/wrapper.py:158]: Any reason for this?
It's rude of libraries to be directly `print`ing - you might screw up a user's intended output (think of programs that pipe their outputs to other programs, it has to be very precise

File: dagshub/upload/wrapper.py [hunk 2]
```diff
@@ -84,6 +84,8 @@ def create_repo(repo_name, is_org=False, org_name="", description="", private=Fa
     repo = res.json()
     return Repo(owner=repo["owner"]["login"], name=repo["name"], token=token, branch="main")
 
+def clean_directory_name(directory: str):
```
Comment (kbolashev) [inline dagshub/upload/wrapper.py:87]: Make it a static function in DataSet, also probably with a _ so it's "private", no reason for it to be in the top level of the module

**Review by kbolashev — CHANGES_REQUESTED** [review]
linter issues btw

**Review by kbolashev — APPROVED** [review]
lgtm 👍

File: dagshub/upload/wrapper.py [hunk 3]
```diff
@@ -130,6 +130,7 @@ def upload_files(self,
 
         if force:
             data["last_commit"] = self._get_last_commit()
+        logger.warning(f'Uploading {len(files)} files to "{self.full_name}"...')
```
Comment (simonlsk) [inline dagshub/upload/wrapper.py:133]: Why is this a warning?

File: dagshub/upload/wrapper.py [hunk 4]
```diff
@@ -153,7 +154,7 @@ def _log_upload_details(self, data, res, files):
             logger.debug(f"Response ({res.status_code})\n")
 
         if res.status_code == 200:
-            logger.info("Upload finished successfully!")
+            logger.warning("Upload finished successfully!")
```
Comment (simonlsk) [inline dagshub/upload/wrapper.py:157]: And here?
@guysmoilov  I saw your comment but I don't think messages like these should be reported as warnings.

**Pull request merged**